packages/core/src/state-management/repository.ts

```diff
@@ -161,6 +161,7 @@ export class Repository {
   }
 
   async init(): Promise<void> {
+    await this.#deps.keyValueStore.init()
```

Comment:
nitpick: I'd probably just move this into `pinStore.open()`.  Not important though

Reply:
No problem changing this but the `anchorRequestStore` below also uses the `keyValueStore`, so it seems a bit fragile to me to rely on the `pinStore` running the initialization for both?

Reply:
oh good call.  Yeah I didn't realize that the PinStore doesn't fully encapsulate the StateStore anymore.  Leaving this here then does indeed seems better